README.md

```diff
@@ -50,6 +50,10 @@ cd ../sonarpedia-text
 java -jar ../../sonar-rule-api/target/rule-api-2.4.0-SNAPSHOT.jar update
 ```
 
+### Create new secrets
+
+See [CONTRIBUTING.md](./CONTRIBUTING.md).
```

Comment:
Is it okay to add this? As far as I understand, we don't accept user contributions for now.